### PR TITLE
Add DRF serializer field

### PR DESCRIPTION
**Pull request by rixx**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -62,3 +62,12 @@ To build a full URL use the ``full`` method like this:
 You can implement the `get_scheme(url)` and `get_hostname(url)` methods on your
 `Url` class to change your default theme and hostname from the urlman defaults
 of `'http'` and `'localhost'`, respectively.
+
+If you use Django RestFramework, you can use ``urlman.UrlManField`` to provide
```
Comment (andrewgodwin) [inline README.rst]: There should be a space between Rest and Framework, and it should be `REST` not `Rest`, I think?

Comment (rixx) [reply README.rst]: Between `pip install djangorestframework` and `from rest_framework import …` I never remember how this thing is even called, but you're of course right.

**Comment by andrewgodwin** [comment]
Hah, yes, I know it was very picky of me, but I guess that's the danger of small pull requests! I'll push out a release for this now before I forget.

**Comment by rixx** [comment]
Picky is good – and thank you for the release. ✨

**Pull request merged**